imap_processing/glows/l0/glows_l0_data.py

```diff
@@ -1,4 +1,4 @@
-from dataclasses import dataclass, fields
+from dataclasses import dataclass
 
 from imap_processing.ccsds.ccsds_data import CcsdsData
 
```

Comment:
I moved the ground software version into `glows/__init__.py` so it could be imported as necessary. The version will be inside the block header, as requested, although while I was looking into your comments, I noticed that I had `ground_software_version` inside `DirectEventL1A` but not `HistogramL1A` so I will fix that. The L0 data classes are only intended for internal use, so they don't store the ground software version, but the version will be inside all the CDF files, as well as in the filename.